Package.swift

```diff
@@ -0,0 +1,68 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "FluentUI",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v10_14),
+    ],
+    products: [
+        .library(
+            name: "FluentUI",
+            type: .static,
+            targets: [
+                "FluentUI"
+            ]
+        )
+    ],
```

Comment:
Does this mean the default export for macOS would be a static library too? Currently downstream we consume it via a framework packed inside the NuGet package, but I have some local code testing how it would look to use it as a static library for macOS instead.

Reply:
Yes, but this only applies to clients that explicitly use SPM to consume the package. The existing framework won't be affected, since our official pipeline will continue to use our existing Xcode projects/schemes/etc.